packages/gatsby-admin/gatsby-config.js

```diff
@@ -1,16 +1,28 @@
 module.exports = {
-  plugins: [{
-    resolve: "gatsby-plugin-react-helmet"
-  }, {
-    resolve: "gatsby-plugin-webfonts",
-    options: {
-      fonts: {
-        google: [{
-          family: `Inter`,
-          variants: [`400`, `600`, `800`]
-        }]
-      }
-    }
-  }],
-  pathPrefix: `/___admin`
-};
\ No newline at end of file
+  plugins: [
+    {
+      resolve: "gatsby-plugin-react-helmet",
+      options: {},
+    },
+    {
+      resolve: "gatsby-plugin-webfonts",
+      options: {
+        fonts: {
+          google: [
+            {
+              family: `Inter`,
+              variants: [`400`, `600`, `800`],
+            },
+          ],
+        },
+      },
+    },
+    {
+      resolve: "gatsby-plugin-create-client-paths",
```

Comment:
Would be cool to use Unified Routes in a future version (if possible). I know @ascorbic is using them in Desktop.